jdi-light-html-tests/src/test/java/io/github/epam/example/common/NumberSelectorExampleTests.java

```diff
@@ -0,0 +1,72 @@
+package io.github.epam.example.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.height;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+import static org.testng.Assert.assertEquals;
+
+public class NumberSelectorExampleTests implements TestsInit {
+
+    String number = "2.1";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4731610)